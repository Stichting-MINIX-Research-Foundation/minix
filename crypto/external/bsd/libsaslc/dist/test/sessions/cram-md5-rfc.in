CRAM-MD5
2
AUTHID tim
PASSWD tanstaaftanstaaf
<1896.697170952@postoffice.reston.mci.net>
