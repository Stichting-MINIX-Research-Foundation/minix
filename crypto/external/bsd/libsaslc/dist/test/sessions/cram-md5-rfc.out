tim b913a602c7eda7a495b4e6e7334d3890
